snowplow_tracker/tracker.py

```diff
@@ -1037,4 +1043,7 @@ def add_emitter(self, emitter: EmitterProtocol) -> "Tracker":
         return self
 
     def get_namespace(self) -> str:
-        return self.standard_nv_pairs["tna"]
+        # This is ignored as we know that the key exists
+        # But can be inferred to be Optional[str] as app_id
```

Comment:
I don't understand this comment – this is not about `app_id`? Is the type conflict that we are ignoring here that the `standard_nv_pairs` may contain int values?

Reply:
So because `app_id` is added to the `standard_nv_pairs` dict [here](https://github.com/snowplow/snowplow-python-tracker/blob/cb7e434be13af1f5dfe5b6b3416d062c477f8de1/snowplow_tracker/tracker.py#L93) with a type of `Optional[str]`, the type for the whole `standard_nv_pairs` dict is inferred to be `dict[str, Optional[str]]`. But, we know that `tna` should always be present in the dict, since `namespace` is a required argument.

This ignores MyPy saying `Incompatible return value type (got "str | None", expected "str")` (Also not really a huge fan of just ignoring it, but that would require changes past just typing, so 🤷)

Reply:
Ah okay, that makes sense!

Reply:
I've clarified the comment a bit for any future Matus' looking at it.